src/runner/testRunnerReporter.js

```diff
@@ -24,18 +25,20 @@ class Reporter {
   added: Array<string>;
   removed: Array<string>;
   interactive: boolean;
+  clrTerm: boolean;
```

Comment:
I wonder why do you use `clearTerminal` everywhere ,but `clrTerm` here?

Reply:
I would have used `clearTerminal` there too, but there is already method of `Reporter` with that name so I can't use it for that property. I could use `_clearTerminal` but that was inconsistent with the other property names ...but so is `clrTerm`. :-| 

Really, knowing that `clearTerminal` was already in use, `clrTerm` was just what came out as I was typing; I didn't give it much more thought. I would gladly change it if you have a different preference.

Reply:
I could set `options` as a class property and read options from there rather than destructuring `options` into separate class properties (we are already using `ReporterOptions` to define the structure and content of `options`). I'm thinking something like this:
```javascript
class Reporter {
  ...
  options: ReporterOptions;
  ...
  constructor(options: ReporterOptions) {
    ...
    // (remove destrucuiring of potions)
    this.options = options;
    // (remove `interactive`, `quite`, and `clrTerm` from contructor)
    this.formatStats = createStatsFormatter(this.options.cwd);
    ...
  }

  logInfo(...args: Array<any>) {
    if (!this.options.quiet) {
      log(...args);
    }
  }

  clearTerminal() {
    if (this.options.clearTerminal && this.options.interactive) {
      process.stdout.write(process.platform === 'win32' ? '\x1B[2J\x1B[0f' : '\x1B[2J\x1B[3J\x1B[H');
    }
  }
  ...
}
```

I think that solution cleaner and more consistent. Thoughts?

If you agree, I'll get it updated right away.

Reply:
@karlwilbur Yup, I agree, looks more consistent

Reply:
I'm going to leave the destructuring of `options` in the constructor for `cwd` and `eventEmitter`; having `this.options.` in front of those `eventEmitter` calls just looks wrong. I'll only destructure those `options` components used in the constructor. I'll update the rest as I described above.